server/src/librelay/relay_server_test.go

```diff
@@ -81,41 +82,41 @@ type TestServer struct {
 	*RelayServer
 }
 
-func (relay *TestServer) Stake(ownerKey *ecdsa.PrivateKey) (err error) {
-	tx, err := relay.sendStakeTransaction(ownerKey)
+func (relay *TestServer) Stake(ownerKey *ecdsa.PrivateKey, stakeAmount *big.Int, unstakeDelay *big.Int) (err error) {
```

Comment:
a. not sure what you gained by removing the fields, if you now need to move them manually around to methods.
b. clearly its not a tiny cleanup, and doesn't fit into the gas-price fix.

Reply:
These methods are just in the tests, they aren't called anywhere else so no reason to keep fields just for methods defined and used only in a test file.